<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationOCS</class>
 <widget class="QWidget" name="SimulationOCS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simulation OCS</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_OCS">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>104</y>
     <width>1127</width>
     <height>95</height>
    </rect>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>800</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>16</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column/>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>924</x>
     <y>234</y>
     <width>107</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Net Disconnected</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>228</y>
     <width>511</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_home_az">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Home AZ</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_home_alt">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Home ALT</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_home_dero">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Home DERO</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton_check_status">
      <property name="text">
       <string>Check status</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButton_connect">
      <property name="text">
       <string>DisConnect</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_point_az">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Point AZ</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_point_alt">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Point ALT</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_start_dero">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Start DERO</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_start_track">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Start TRACK</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_ocs_selfcheck">
      <property name="text">
       <string>OCS Selfcheck</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_stop_az">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Stop AZ</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_stop_alt">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Stop ALT</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_stop_dero">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Stop DERO</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_stop_track">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Stop TRACK</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButton_guidingStar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Guiding star</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>228</y>
     <width>160</width>
     <height>123</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_Az_point">
      <property name="text">
       <string>180.000</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_Alt_point">
      <property name="text">
       <string>45.000</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_Az_guiding">
      <property name="text">
       <string>1.00</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineEdit_Alt_guiding">
      <property name="text">
       <string>0.50</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Point</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Guiding</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Az</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Alt</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Dero</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_Dero_point">
      <property name="text">
       <string>0.000</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="lineEdit_Dero_guiding">
      <property name="text">
       <string>2.00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget_OCSout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>8</y>
     <width>1127</width>
     <height>95</height>
    </rect>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>800</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>16</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
